vkcs/vpnaas/resource_site_connection.go

```diff
@@ -185,6 +193,12 @@ func resourceSiteConnectionCreate(ctx context.Context, d *schema.ResourceData, m
 
 	adminStateUp := d.Get("admin_state_up").(bool)
 	initiator := resourceSiteConnectionInitiator(d.Get("initiator").(string))
+	var trafficSelector *isiteconnections.TrafficSelectorEPMergeExt
+	if networking.GetSDN(d) == inetworking.SprutSDN {
```

Comment:
does it work with the only sprut sdn in a project and no sdn property is set?

Reply:
I can make it so that this parameter can only be set when sdn is set.  Or I may replace check  to sdn != neutron and user get an error when try to create.

Reply:
add RequiredWith sdn and set only for sprut